lib/ansible/modules/windows/win_file.py

```diff
@@ -47,19 +47,20 @@
         the C(path) will be deleted and set as a new junction point.
       - If state=C(hard) and C(path) exists and is not a hard link of C(src),
         the C(path) will be deleted and set as a new hard link.
-    choices: [true, false]
+    type: bool
+    default: no
```

Comment:
This should be 'no', because otherwise it will be a boolean in the documentation and considered False.
That is why `type: bool` translates to `choices: [ 'no', 'yes' ]`.